WalletWasabi.Tests/UnitTests/WabiSabi/Backend/PhaseStepping/StepTransactionSigningTests.cs

```diff
@@ -243,6 +230,55 @@ public async Task TimeoutSufficientPeersAsync()
 		await arena.StopAsync(token);
 	}
 
+	[Fact]
+	public async Task AliceWasNotReady()
+	{
+		using CancellationTokenSource cancellationTokenSource = new(TestTimeout);
+		var token = cancellationTokenSource.Token;
+
+		WabiSabiConfig cfg = new()
+		{
+			MaxInputCountByRound = 2,
+			MinInputCountByRoundMultiplier = 0.5,
+			TransactionSigningTimeout = TimeSpan.Zero,
+			OutputRegistrationTimeout = TimeSpan.Zero,
+			FailFastTransactionSigningTimeout = TimeSpan.Zero,
+			MaxSuggestedAmountBase = Money.Satoshis(ProtocolConstants.MaxAmountPerAlice)
+		};
+		var (keyChain, coin1, coin2) = WabiSabiFactory.CreateCoinKeyPairs();
+
+		var mockRpc = WabiSabiFactory.CreatePreconfiguredRpcClient(coin1.Coin, coin2.Coin);
+		mockRpc.Setup(rpc => rpc.SendRawTransactionAsync(It.IsAny<Transaction>(), It.IsAny<CancellationToken>()))
+			.ThrowsAsync(new RPCException(RPCErrorCode.RPC_TRANSACTION_REJECTED, "", null));
```

Comment:
We need to remove the dependency with Moq (see: https://github.com/zkSNACKs/WalletWasabi/pull/11249).
After you rebase (or merge) you will see that now `CreatePreconfiguredRpcClient` returns a `MockRpcClient` instance instead of an `IMock<IRpcClient>`. Please use that.